src/R/Support/Impl/Help/IntelliSenseRSession.cs

```diff
@@ -101,6 +101,21 @@ public sealed class IntelliSenseRSession : IIntellisenseRSession {
             }
         }
 
+        public async Task StopSessionAsync() {
+            var token = await _lock.ResetAsync();
+            try {
+                if (!_sessionProvider.HasBroker) {
```

Comment:
Why do you care if `_sessionProvider.HasBroker == true`? I don't think its even possible to have running session without a broker.